Fails on 4.x versions of Philhower Core - requires downgrade to work

HW - Adafruit Feather RP2040 with DVI Output Port - brand new
Steps: 

- Install Raspberry Pi Pico/RP2040 by Earle F Philhower, III - current version 4.1.1
- Install Adafruit fork of PicoDVI using library manager
- Load example: Examples->PicoDV - Adafruit Fork->8bit_single_buffer
- Compiling give the following error:

```
c:\Users\shane\codes\sketches\libraries\PicoDVI_-_Adafruit_Fork\src\libdvi\dvi.c: In function 'dvi_dma_irq_handler':
c:\Users\shane\codes\sketches\libraries\PicoDVI_-_Adafruit_Fork\src\libdvi\dvi.c:194:68: error: 'dma_debug_channel_hw_t' has no member named 'tcr'
  194 |                 while (dma_debug_hw->ch[inst->dma_cfg[i].chan_data].tcr != inst->timing->h_active_pixels / DVI_SYMBOLS_PER_WORD)
      |                                                                    ^

exit status 1

Compilation error: exit status 1 
```

I know this has worked in the past, because I have several of these Feather DVIs and they's worked great. Doing a search for `dma_debug_hw` showed me it is a part of Philhower core, so I was able to temporarily unblock myself by downgrading the Pi Pico/RP2040 board manager to 3.9 since it seems the first broken version was 4.0.

It would be great if we could use the PicoDVI library on current versions of Pi Pico/RP2040.

--- a/library.properties
+++ b/library.properties
@@ -1,5 +1,5 @@
 name=PicoDVI - Adafruit Fork
-version=1.1.0
+version=1.2.0
 author=Luke Wren (Wren6991)
 maintainer=Adafruit <info@adafruit.com>
 sentence=Arduino library for RP2040 DVI output, based on PicoDVI

--- a/src/libdvi/dvi.c
+++ b/src/libdvi/dvi.c
@@ -191,7 +191,11 @@ static void __dvi_func(dvi_dma_irq_handler)(struct dvi_inst *inst) {
 	// Make sure all three channels have definitely loaded their last block
 	// (should be within a few cycles of one another)
 	for (int i = 0; i < N_TMDS_LANES; ++i) {
+#if PICO_SDK_VERSION_MAJOR == 2
+		while (dma_debug_hw->ch[inst->dma_cfg[i].chan_data].dbg_tcr != inst->timing->h_active_pixels / DVI_SYMBOLS_PER_WORD)
+#else
 		while (dma_debug_hw->ch[inst->dma_cfg[i].chan_data].tcr != inst->timing->h_active_pixels / DVI_SYMBOLS_PER_WORD)
+#endif
 			tight_loop_contents();
 	}
 

